### ISSUE TITLE
Automatic map votes appear to be broken on live.

### ISSUE THREAD

**Issue by ArcaneMusic** [tgstation/tgstation#64963]
## Testmerges:

None active.

## Reproduction:

Wait for the passive map vote to be called by the game, it will produce an empty map vote as seen below:
![image](https://user-images.githubusercontent.com/41715314/154366330-84806cd5-0196-43bb-aa18-13c986c229f8.png)
Unsure if this is a server issue or if there were recent changes to how map votes are held codewise.

**Comment by MrMelbert** [comment]
![image](https://user-images.githubusercontent.com/51863163/154375957-2f5abf43-0efd-4d3a-bf4f-2a9534dc11ee.png)

I checked the map config when this happened in game and this is what it appeared to be. Top is max players, bottom is min players.

**Comment by MrMelbert** [comment]
tl;dr I think SSPersistance needs to not block the only votable maps
